README.md

```diff
@@ -22,7 +24,9 @@ diener update --substrate --branch polkadot-v0.9.18
 cargo build --release
 ```
 
-Place the binary into `../../bin/polkadot-0.9.18-sudo`
+Run:
+`cp target/release/polkadot ../../bin/polkadot-0.9.18`  
```

Comment:
`-sudo` prefix is not needed?

Reply:
with sudo it crashed because it was looking for the one without sudo, so i renamed it

Reply:
Not ok. Kusama-local needs a patch as documented above. I'd like this patch to be explicitly visible

Reply:
i dont understand. if i run 
`node ../polkadot-launch/dist/cli.js ./polkadot-launch/launch-rococo-local-with-launch.json`

it get
```
🧹 Resolving parachain id...
2022-04-16 08:24:56 Building chain spec    
  ✓ Read parachain id for ../target/release/encointer-collator: 1003
Relay chain binary does not exist:  /Users/pigu/Dropbox/DATA/Documents/solidbit/kunden/encointer/code/bin/polkadot-0.9.18
```
could it be that the previous author of this readme is making assumptions about the `bin` folder (for example that a `polkadot-0.9.18` binary should already exist there) that are not documented?

Reply:
yes, I make these asumptions and the patched binary must be named with -sudo suffix. Without this patch, the kusama setup won't work locally

Reply:
ok, i will update the readme accordingly